reflections/2020-05-22.md

```diff
@@ -0,0 +1,29 @@
+# Reflections
+
+## Participation
+
+- 3 groups 6-8 people in size.
+- Each group had a brainhack committee member and a python/programming enthusiast/expert
+
+## What Happened
+
```

Comment:
There seemed to be questions on how to interpret different colors (something specific to colab but not necessarily a feature only found in colab) for say 'string' versus string. There was also questions about how a variables value can be modified. For example: weight_kg = 100, what happens to weight_kg when I run -- weight_kg * 2.2 = weight_lb?